app/styles/snippet-plugin.scss

```diff
@@ -164,3 +141,62 @@ div[typeof='besluitpublicatie:Documentonderdeel']
     }
   }
 }
+
+.say-snippet-placeholder {
+  display: flex;
+  flex-direction: row;
+  padding: $au-unit-small $au-unit-small * 1.5;
+  background-color: #fff7bb;
+  border-radius: var(--au-radius);
+  border: 0.1rem solid #e5d34f;
+}
+
+.say-snippet-placeholder__icon {
+  background-color: #ffa10a;
+  color: black;
+  height: $au-unit - 0.1rem; // compensate for visual distortion of perfect circle
+  width: $au-unit;
+  margin-right: $au-unit * 0.5;
+  border-radius: $au-unit-large;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+}
+
+.say-snippet-placeholder__content {
+  margin-top: 0 !important;
```

Comment:
Is it possible not to add important here? maybe with more specifity, if not is okay

Reply:
Waiting on @elpoelma's fix: https://github.com/lblod/ember-rdfa-editor/pull/1226

Reply:
Was able to fix it, it was also still another issue in the end.